opendbc/car/subaru/carcontroller.py

```diff
@@ -32,30 +33,47 @@ def update(self, CC, CS, now_nanos):
 
     # *** steering ***
     if (self.frame % self.p.STEER_STEP) == 0:
-      apply_torque = int(round(actuators.torque * self.p.STEER_MAX))
-
-      # limits due to driver torque
+      if self.CP.flags & SubaruFlags.LKAS_ANGLE:
+        actual_steering_angle_deg = CS.out.steeringAngleDeg
+        desired_steering_angle_deg = actuators.steeringAngleDeg
+
+        apply_steer = apply_std_steer_angle_limits(
+          desired_steering_angle_deg,
+          self.apply_steer_last,
+          CS.out.vEgoRaw,
+          actual_steering_angle_deg,
```

Comment:
```suggestion
          CS.out.steeringAngleDeg,
```

easier to read to not have intermediary variables around here